.github/workflows/amend-user-repository-access.yml

```diff
@@ -1,10 +1,8 @@
 name: Amend users access to a repository
 
-# Allows you to run this workflow manually from the Actions tab
 on:
   schedule:
-    # At the end of every day
-    - cron: "0 6 * * *"
+    - cron: "0 3 * * *"
```

Comment:
The job failed this moring, and yesterday when I manually ran it. 

Seems to hit the RateLimit twice and exit. Assuming that since this job and the one for MoJAS are running at the same time, we're increasing the chances of hitting rate limits more often. 

So hoping that moving this back a bit mitigates the problem 🤞

Reply:
[REF](https://github.com/ministryofjustice/operations-engineering/actions/runs/4361639464/jobs/7625712121#step:5:55)

Reply:
Agree moving the run time